wiki/mapping/mediocre-map-assistant.md

```diff
@@ -64,7 +48,40 @@ instead of
 `C:\Steam\steamapps\common\Beat SaberBeat Saber_Data\CustomLevels`
 :::
 
-#### Check the WIP File Path
+### Linux Installation
+1. Download MMA2.zip from [GitHub](https://github.com/Assistant/MediocreMapAssistant2/releases/latest) 
+:::danger NOTE
+The download link for MMA2 is currently down. We are currently working to resolve this issue. Please hop over to the `#mapping-discussion` channel in the [BSMG Discord](https://discord.gg/beatsabermods) for more info. 
+:::
+
+2. Unzip the file and place the extracted folder wherever you like on your hard drive.
+3. Open Steam. At the top of the window, go to `Steam > Settings > Steam Play`. In the `Advanced` section, check `Enable Steam Play for all other titles` if it isn't already. Restart Steam if you are prompted to do so.
+3. On the bottom right of Steam's main window, click `Add a Game`, then `Add a Non-Steam Games`. A window opens, click `Browse` at the bottom. Select `All Files` as the file type. Find where `MediocreMapAssistant2.exe` is, and add it.
+4. Search for `Proton` in the Steam search bar, then install the latest version.
+5. Right-click the `MediocreMapAssistant2.exe` shortcut, and open the Properties menu. Check `Force the use of a specific Steam Play compatibility tool`, and select the latest version of Proton in the dropdown menu. At this step, you might also want to rename and add an icon to the shortcut as Linux cannot read the built in logo for MMA2.
+6. You can now use MMA2 by starting it through Steam. You can also create a shortcut by right-clicking the MMA2 shortcut in Steam, then under the `Manage` section clicking `Add Desktop Shortcut`.
+
+::: warning 
+It is **VERY** important to unzip the file in step 2 and only run the exe in the new folder instead of the one inside the zip as not doing so will cause issues with map creation and editing!
+:::
+
+#### Linux First Time Setup
+The first time you run MMA2 you will need to direct it to the location of two folders: `CustomLevels` and `CustomWIPLevels`. You have several options available to you:
+
+**If you have Beat Saber installed on the computer:**
```

Comment:
BeatSaber does not natively work under Linux, since it's VR i doubt it will run well with Proton (but I can't test this since im a quest owner).

Reply:
Edit: checked Beat Saber on ProtonDB, it's classed as Platinium (Works flawlessly at first launch), so there might actually be people concerned by this.